From dab05f848a3a894a7849fb92d5c4d47aa6b84813 Mon Sep 17 00:00:00 2001
From: Michal Galka <michal.galka@collabora.com>
Date: Thu, 17 Jun 2021 14:26:57 +0200
Subject: [PATCH] STAGING replace bootrr commit shash with staging.kernelci.org

For staging builds buildroot should use HEAD of the staging.kernelci.org
branch instead of a specific commit id. This change should be applied
only for staging build.

Signed-off-by: Michal Galka <michal.galka@collabora.com>
---
 package/bootrr/bootrr.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/package/bootrr/bootrr.mk b/package/bootrr/bootrr.mk
index e10abbc237..99c43278c3 100644
--- a/package/bootrr/bootrr.mk
+++ b/package/bootrr/bootrr.mk
@@ -9,6 +9,10 @@ BOOTRR_VERSION = f15e3f8b13254bcf586130b2603f1bdddfcf630d
 BOOTRR_SITE_METHOD = git
 BOOTRR_LICENSE = LGPL-2.1+
 
+
+BOOTRR_VERSION = staging.kernelci.org
+BR_NO_CHECK_HASH_FOR += $(BOOTRR_SOURCE)
+
 define BOOTRR_INSTALL_TARGET_CMDS
 	DESTDIR=$(TARGET_DIR)/opt/bootrr $(MAKE) -C $(@D) install
 endef
-- 
2.25.1

